F1/F1/Login/LoginViewModel.swift

```diff
@@ -8,16 +8,17 @@
 import Foundation
 
 class LoginViewModel {
+
+    // MARK: Variables
     private let username = "Admin"
     private let password = "Password"
 
-    func validDetails(givenUsername: String, givenPassword: String) -> Bool{
+    // MARK: Functions
+    func validDetails(givenUsername: String, givenPassword: String) -> Bool {
         if givenUsername==username && givenPassword==password {
             return true
```

Comment:
try avoid these if something return true else return false
You can make it so much cleaner and just say 
return givenUsername == username && givenPassword == password